.gitignore

```diff
@@ -16,7 +16,6 @@ __media
 _example.js
 codegen.yaml
 schema.graphql
-
```

Comment:
?

Reply:
it's after yarn install, anyway someone will merge this sooner or later :)

Reply:
I added submodules here, then removed (not needed actually), and looks like I just removed meaningless empty row too.